Emergency power unit

DebugOk opened the pull request:
## About the PR
Adds the Emergency Power Update, part of the "Fucking over salvage part 2" update.

TODO:
- [x] Get a proper sprite
- [x] Decide if ~15 minutes of power is enough for the high capacity version

## Why / Balance
I'm going to mess up their starting shuttle even more, but I don't want it running out of power too fast

Its literally just an SMES that cant be charged

## Media
![Content Client_2ka3iLXIYG](https://github.com/DeltaV-Station/Delta-v/assets/49997488/c0111689-9e8e-48d3-a018-391d263aa15d)
![Content Client_bGUc3NiXIb](https://github.com/DeltaV-Station/Delta-v/assets/49997488/0122b4d7-4fe1-4518-9eba-b64628223789)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

DebugOk committed:
Emergency power unit

deltanedas commented:
> Its literally just an SMES that cant be charged

mining station has this too for reference (unless you stole from there already :trollface:)

DebugOk commented:
> > Its literally just an SMES that cant be charged
> 
> mining station has this too for reference (unless you stole from there already :trollface:)

Its pretty much the same idea :blunt:

DebugOk committed:
Add texture

DebugOk committed:
Change prototype

DebugOk commented:
Should be good to go

DebugOk committed:
Prevent deconstruction of the EPU

Surely this wont break anything...
Cant wait for https://github.com/space-wizards/space-station-14/pull/24654 to be merged

DebugOk commented:
Probably gonna self merge tomorrow if its not reviewed before then:tm:

deltanedas commented on Resources/Textures/DeltaV/Structures/Power/epu.rsi/meta.json:
```diff
@@ -0,0 +1,77 @@
+    {
+      "name": "smes"
+    },
+		{
```
troll indent

DebugOk replied:
Real

DebugOk commented:
Gotta figure out how to unfail the test first lol, I wonder if theres a better way to prevent deconstruction

deltanedas commented:
mining station just has it not give you a board, so you can scrap it for steel and lv if you really dont need it

DebugOk commented:
> mining station just has it not give you a board, so you can scrap it for steel and lv if you really dont need it

They completely copied the smes prototype, which I wanted to avoid if possible. But it seems like I'll have to if I want it to not give a board

VMSolidus commented:
So I had originally been utterly confused why we seemingly want to screw over salvage players. And after playing cargo for a few rounds, I now understand that it's because they genuinely deserve it. This didn't have my approval last night, but it does now. Please do, you now have my support to do everything in your power to screw them over as much as possible.

DebugOk commented:
> So I had originally been utterly confused why we seemingly want to screw over salvage players. And after playing cargo for a few rounds, I now understand that it's because they genuinely deserve it. This didn't have my approval last night, but it does now. Please do, you now have my support to do everything in your power to screw them over as much as possible.

Luckily I don't need approval and it would've been merged anyways, salvage will keep getting nerfs until they stop powergaming the shit out of everything they can get their hands on

VMSolidus commented:
Good. They deserve it.
